### PR TITLE
allows to use custom cube properties to docker-compose

### PR DESCRIPTION
**Pull request by lordofthejars**
allows to use custom cube properties to docker-compose defined containers.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -527,6 +530,31 @@ beforeStop:
...
+        strategy: polling
+      beforeStop:
+        - copy:
+            from: /test
+            to: /tmp";
+</property>
+----
+
+Previous snippet would override `await` strategy to polling and set a `beforeStop` event in cube container named `tomcat` defined in a previous docker-compose definition.
+Also notice that the `image` property here does nothing, since ony CUBE custom properties are overriden.
```
Comment (aslakknutsen) [inline README.adoc]: Shouldn't this be removed from the example and only explained in text? Are there other Cube specific configurations that we should mention besides Await and BeforeStop ?

Comment (lordofthejars) [inline README.adoc]: You mean `image`. Yes I though the same but I found a way to enforce the message to reader that only custom properties are updated.

Comment (aslakknutsen) [inline README.adoc]: Yeah, I understand the point. Just not sure ppl actually read the text :)

Comment (lordofthejars) [inline README.adoc]: Ok I remove then.

**Comment by lordofthejars** [comment]
Fixes #254

**Pull request merged**